backend/core/src/email/email.service.ts

```diff
@@ -186,19 +199,41 @@ export class EmailService {
   }
 
   private async loadTranslations(jurisdiction: Jurisdiction | null, language: Language) {
-    const jurisdictionalTranslations = await this.translationService.getTranslationByLanguageAndJurisdictionOrDefaultEn(
-      language,
-      jurisdiction ? jurisdiction.id : null
-    )
-    const genericTranslations = await this.translationService.getTranslationByLanguageAndJurisdictionOrDefaultEn(
-      language,
+    let jurisdictionalTranslations: Translation | null,
+      genericTranslations: Translation | null,
+      jurisdictionalDefaultTranslations: Translation | null
+
+    if (language != Language.en) {
+      if (jurisdiction) {
+        jurisdictionalTranslations = await this.translationService.getTranslationByLanguageAndJurisdictionOrDefaultEn(
```

Comment:
I think we can dry this up with the block on 220 so we don't have to call it twice

Reply:
Hmm, maybe we should try to clean this up on the translationService level, because the issue I was seeing before was a lack of proper fallback — aka it wasn't going through the flow of 

1. Look up specific jurisdiction and specific language
2. Not found? look up specific jurisdiction and English language
3. Not found? look up non-jurisdiction and specific language
4. Not found? finally look up non-jurisdiction and English language

If we try to DRY up this section we lose some of those fallbacks. Thoughts?

Reply:
Good to manage some cleanup within the service later 👍